src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
```

Comment:
* Kindly fix this failing test, I think its the default test that came with installing react 
![image](https://user-images.githubusercontent.com/52497246/194524487-425bd647-bffc-4d24-8ad9-b2ec9f7b4140.png)